doc/src/manual/constructors.md

```diff
@@ -585,27 +585,25 @@ defined on `Bool` and various supertypes of it:
 
 ```julia-repl
```

Comment:
Isn't it easier to use [`@repl`](https://documenter.juliadocs.org/stable/man/syntax/#@repl-block) instead of manually updating this every time?